### PR TITLE
Reorder messages

### PR DESCRIPTION
**Pull request by hieronx**

### CODE REVIEW

File: src/libraries/MessagesLib.sol [hunk 1]
```diff
@@ -13,70 +13,69 @@ library MessagesLib {
...
-        /// 9 - Increase an investment order by a given amount
+        /// 18 - Update the member list of a tranche token with a new member
+        UpdateMember,
+        /// 19 - Freeze tranche tokens
+        Freeze,
+        /// 20 - Unfreeze tranche tokens
+        Unfreeze,
+        /// --- Investment Manager ---
+        /// 21 - Increase an investment order by a given amount
         IncreaseInvestOrder,
```
Comment (wischli) [inline src/libraries/MessagesLib.sol:62]:
```suggestion
        RequestDeposit,
```

File: src/libraries/MessagesLib.sol [hunk 2]
```diff
@@ -13,70 +13,69 @@ library MessagesLib {
...
-        /// 10 - Decrease an investment order by a given amount
-        DecreaseInvestOrder,
-        /// 11 - Increase a Redeem order by a given amount
+        /// 22 - Increase a Redeem order by a given amount
         IncreaseRedeemOrder,
```
Comment (wischli) [inline src/libraries/MessagesLib.sol:64]:
```suggestion
        RequestRedeem,
```

File: src/libraries/MessagesLib.sol [hunk 3]
```diff
@@ -13,70 +13,69 @@ library MessagesLib {
...
-        FulfilledCancelRedeemRequest,
-        /// 17 - Executed Collect Invest
+        /// 23 - Executed Collect Invest
         FulfilledDepositRequest,
-        /// 18 - Executed Collect Redeem
+        /// 24 - Executed Collect Redeem
         FulfilledRedeemRequest,
-        /// 19 - Cancel an investment order
+        /// 25 - Cancel an investment order
         CancelInvestOrder,
```
Comment (wischli) [inline src/libraries/MessagesLib.sol:70]:
```suggestion
        CancelDepositRequest,
```

File: src/libraries/MessagesLib.sol [hunk 4]
```diff
@@ -13,70 +13,69 @@ library MessagesLib {
...
-        /// 20 - Cancel a redeem order
+        /// 26 - Cancel a redeem order
         CancelRedeemOrder,
```
Comment (wischli) [inline src/libraries/MessagesLib.sol:72]: CancelRedeemRequest

**Review by wischli — COMMENTED** [review]
Four message names were not updated

File: src/libraries/MessagesLib.sol (hunk 4 above)
Comment (wischli) [inline src/libraries/MessagesLib.sol:72]:
```suggestion
        CancelRedeemRequest,
```

File: src/libraries/MessagesLib.sol (hunk 1 above)
Comment (hieronx) [reply src/libraries/MessagesLib.sol:62]: True, good catch, thanks!

Comment (hieronx) [reply src/libraries/MessagesLib.sol]: @wischli changed to `DepositRequest` (and likewise for `RedeemRequest`) to be more consistent with the other messages

File: src/interfaces/gateway/IGasService.sol [hunk 5]
```diff
@@ -36,6 +36,10 @@ interface IGasService {
     /// @return The current price
     function tokenPrice() external returns (uint256);
 
+    /// @notice Executes a message from the gateway
+    /// @dev    The function can only be executed by the gateway contract.
+    function handle(bytes calldata message) external;
+
     /// @notice Updates the gas price on Centrifuge Chain
     /// @dev    This is called through PoolManager and the update
```
Comment (peculiarity) [inline src/interfaces/gateway/IGasService.sol:44]:
```suggestion
    /// @dev    
```

Comment (hieronx) [reply src/interfaces/gateway/IGasService.sol]: Fixed!

**Review by ilinzweilin — APPROVED** [review]

**Pull request merged**